xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -121,11 +122,12 @@ namespace XBMCAddon
       /// @param      offlineKey      bool - Specifes the type of the request.
```

Comment:
Misspelling: specifes -> specifies

I grepped the codebase, the is the only instance of `specifes`. Is there a plugin, similar to clang-tidy, that can check spelling? There's nothing sexier than pages and pages of endless documentation with no unsightly errors.

Reply:
This was the better option I could find so far: https://github.com/louis-langholtz/pyspellcode

Reply:
I gave the tool above a try but it didn't find any typos in the python docs. Even the "specifes" one was not found :/

Reply:
haha sucky